webdriver-spec.html

```diff
@@ -504,6 +505,68 @@
 </section> <!-- /Conformance -->
 
 <section>
+<h2>Introduction</h2>
+
+<p><i>This section is non-normative</i>
+
+<section>
+<h3>Design Notes</h3>
+
+<p>This specification attempts to follow a number of design goals:
+
+<section>
+<h4>Compatibility with existing Selenium WebDriver functionality</h4>
+
+<p>This specification is derived from the popular
+ <a href=https://seleniumhq.org>Selenium WebDriver</a> browser
+ automation framework. Selenium is a long-lived project, and due to
+ its age and breadth of use it has a wide range of expected
+ functionality. This specification uses these expectations to inform
+ its design. Where improvements or clarifications have been made, they
+ have been made with care to allow existing users Selenium WebDriver
+ to avoid unexpected breakages.
+</section>
+
+<section>
+<h4>Suitability for use end-user browser automation needs</h4>
+
+<p>It is expected that the largest group of users of this
+ specification are going to be software developers and testers writing
+ automated tests and other tooling, such as monitoring or load
```

Comment:
I would rephrase this slightly by removing the word “expected” and say that this is the _intended_ user group.